### DISCUSSION TITLE
v3.0.32

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#8044] [General]
## What's Changed
* Truncate long badge label by @qrasa1 in https://github.com/filamentphp/filament/pull/8018
* Update Tables > HasActions to not override configured actionsPosition unless provided by @tiagof in https://github.com/filamentphp/filament/pull/8023
* FIX: Usage of HTML Titles by @bernhardh in https://github.com/filamentphp/filament/pull/8025

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.0.31...v3.0.32

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.0.32'>v3.0.32</a>.</em>

**Comment by atmonshi** [comment]
When updating to this version, all form components disappear when having a `Translatable` trait!
I get an empty page.

<img width="730" alt="Screenshot 2023-08-24 at 8 34 44 PM" src="https://github.com/filamentphp/filament/assets/1952412/f094b4ac-0f72-4f73-9412-17cd0e51e585">

downgrading to v3.0.31 will work fine

**Reply by MPTA** [reply]
Exactly my problem

**Reply by atmonshi** [reply]
yup

**Reply by MPTA** [reply]
i fixed it just temporary , i commented line code `$this->fillFormWithDataAndCallHooks($data);`  line 22 of the file located 
/vendor/filament/spatie-laravel-translatable-plugin/src/Resources/Pages/Concerns/HasTranslatableFormWithExistingRecordData.php

**Reply by danharrin** [reply]
Should be fixed in .33

**Reply by atmonshi** [reply]
@MPTA this will not fill the form with the data in edit :)

**Comment by MPTA** [comment]
hi, after i updated to v3.0.32 , when add Translatable Trait in the EditPage of resource, form will hide and n any element shown
<img width="1264" alt="Screen Shot 1402-06-02 at 21 05 55" src="https://github.com/filamentphp/filament/assets/40070445/3140cc9f-806b-4e3a-b8a3-ae3bc23271ce">
<img width="631" alt="Screen Shot 1402-06-02 at 21 06 17" src="https://github.com/filamentphp/filament/assets/40070445/27147174-87f8-4cef-bd1d-74992a37b003">

**Comment by MPTA** [comment]
> ## What's Changed
> 
>     * Truncate long badge label by @qrasa1 in [Truncate long badge label #8018](https://github.com/filamentphp/filament/pull/8018)
> 
>     * Update Tables > HasActions to not override configured actionsPosition unless provided by @tiagof in [Update Tables > HasActions to not override configured actionsPosition unless provided #8023](https://github.com/filamentphp/filament/pull/8023)
> 
>     * FIX: Usage of HTML Titles by @bernhardh in [FIX: Usage of HTML Titles #8025](https://github.com/filamentphp/filament/pull/8025)
> 
> 
> **Full Changelog**: [v3.0.31...v3.0.32](https://github.com/filamentphp/filament/compare/v3.0.31...v3.0.32)
> _This discussion was created from the release [v3.0.32](https://github.com/filamentphp/filament/releases/tag/v3.0.32)._

Has the problem of not displaying the EDIT form fields in version .34 when adding the translatable Trait been solved?

**Reply by danharrin** [reply]
it should have been fixed in .33
